components/pkg-export-docker/src/docker.rs

```diff
@@ -70,10 +70,7 @@ impl<'a> DockerBuilder<'a> {
     /// * If building the Docker image fails
     pub fn build(self) -> Result<DockerImage> {
         let mut cmd = docker_cmd();
-        cmd.current_dir(self.workdir)
-           .arg("build")
-           .arg("--force-rm")
-           .arg("--no-cache");
```

Comment:
I did a lot of spelunking to determine why, and it appears to have been there since the beginning, with no comment.

Based on https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache, however, I believe that we are safe in not using `--no-cache`. If we had commands like `RUN apt-get -y update` we'd need it, but all the commands we use seem to do the right thing.

Of course, I'm open to second opinions on that! :sweat_smile: